pkg/ocm/ocm.go

```diff
@@ -224,6 +233,33 @@ func (c *SdkClient) PostLimitedSupportReason(limitedSupportReason *LimitedSuppor
 	return nil
 }
 
```

Comment:
I wonder if we can make this cleaner. An error is probably not the ideal way to convey an expected failure.

Reply:
Same :D Let me see...